src/main/kotlin/model/UserPreferenceApi.kt

```diff
@@ -0,0 +1,38 @@
+package uk.gov.justice.hmpps.architecture
+
+import com.structurizr.model.Container
+import com.structurizr.model.Model
+import com.structurizr.model.SoftwareSystem
+import com.structurizr.view.ViewSet
+import uk.gov.justice.hmpps.architecture.annotations.Tags
+
+class UserPreferenceApi private constructor() {
+  companion object : HMPPSSoftwareSystem {
+    lateinit var system: SoftwareSystem
+    lateinit var api: Container
+
+    override fun defineModelEntities(model: Model) {
+      system = model.addSoftwareSystem(
+        "User preference api",
+        "Stores preference information associated with a HMPPs Auth User"
+      )
+
+      api = system.addContainer("API", "API", "Kotlin + Spring Boot").apply {
+        url = "https://github.com/ministryofjustice/hmpps-user-preferences"
```

Comment:
🤔 This one didn't show up in https://structurizr.com/share/56937/documentation#/:Published%20APIs, despite having a badge